README.md

```diff
@@ -647,10 +647,71 @@ type C = Omit<A, B>;
 
 However, Flow implementation is stricter in this case, as B have a property that A does not have, it would rise an error. In Typescript, however, they would be ignored. 
 
+## Private properties in classes
+
+### flow
```

Comment:
The example below isn't `flow` specific. It works the same way as in TypeScript.

Reply:
If this could be specified as "same syntax" (without separation), then I'll accept.